iam/README.rst

```diff
@@ -7,9 +7,9 @@ Python Client for IAM Service Account Credentials API (`Alpha`_)
 - `Product Documentation`_
 
 .. _Alpha: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/README.rst
-.. _IAM Service Account Credentials API: https://cloud.google.com/iamcredentials
-.. _Client Library Documentation: https://googlecloudplatform.github.io/google-cloud-python/latest/iamcredentials/usage.html
-.. _Product Documentation:  https://cloud.google.com/iamcredentials
+.. _IAM Service Account Credentials API: https://cloud.google.com/iam
```

Comment:
Should this link be specific to the [IAM credentials docs](https://cloud.google.com/iam/credentials/reference/rest/)?

Reply:
just generalize this to be 'iam'. it isn't clear if we are going to bring other services yet, but it seems likely.